### DISCUSSION TITLE
[Bug]: XDebug in Herd Corrupting SQlite Database

### DISCUSSION THREAD

**Discussion by ahajicek07** [beyondcode/herd-community#1048] [General]
### Platform

Windows

### Operating system version

Windows 11 Pro 10.0.22631 Build 22631

### System architecture

Windows

### Herd Version

1.11.0

### PHP Version

PHP 8.3

### Bug description

Xdebug is somehow corrupting my local SQlite databases. Randomly it would error out, I'd see the error and then my SQlite Database was corrupted. 

Its happened 2x to me in a couple days of working locally on my computer. 

Just now, I ran Xdebug, it automatically ran all my pest tests, then logged me out of my local filamentPHP backend and when I check out the database with Tableplus is says 'database disk image is malformed.' 

### Steps to reproduce

Randomly running Xdebug.

**Comment by mpociot** [comment]
I converted this into a discussion, as this seems to be more of an issue with either Xdebug or SQLite in general, but nothing Herd related. Maybe it's worth opening an issue upstream on the xdebug repository if you find a way to reproduce this.

**Comment by ahajicek07** [comment]
No problem, I'll try to get more relevant information if you just want to leave this discussion open for now. I have to finish this project in a big rush and it was really slowing me down so I just unenabled Xdebug in my php.ini so it would stop happening. I'll poke around later once I finish this project and see what I can find out more.
